src/pages/authorization/authorizing-api-calls/index.mdx

```diff
@@ -793,6 +793,16 @@ To achieve that, make the same exact same request as in [Case: New customer](#ca
 
 Your `entity_id`, or any other customer identity data, won' be overwritten. The web browser will attach cookies to the server allowing it to recognize you as an existing customer. You'll get a new access token, and cookies' lifetime will be extended, too. They will be valid for 2 years from your last request to `https://accounts.livechat.com/customer/token`.
 
+
+### Case: Existing customer chatting in different groups
+
+Use this option to generate a new access token for an already existing customer, but with a separate identity for another group. (This requires the `chat_between_groups` license property to be set to `false`.)
```

Comment:
something weird is with the brackets `()` - do we need them?

Reply:
I would also add info that you should make the same request with the same parameters as in the previous cases (or in the first case: new customer) but only different URL

Reply:
As for brackets, this is the correct spelling and punctuation when there's a full sentence in them :D Can we get rid of them? Maybe we should. Some say that if something's in brackets, it's not important enough to be in the docs :P

Reply:
Also, this description mentions only existing customers, but @bkubiak mentioned it applies to new ones as well. So maybe we should edit the "already existing customer" part

Reply:
hopefully improved in d9493629